tests/test_integration.py

```diff
@@ -8,3 +9,13 @@ def test_health_check():
     response = client.get("/health/")
     assert response.status_code == 200
     assert response.json() == {"status": "OK"}
+
+def test_output_model(): 
+    response = client.get("/api/HondaCivic")
+    assert response.status_code == 200
+    mtcar.model_validate(response.json())
+
+#def test_output_model(): 
+  #  response = client.get("/api/dataset")
+ #   assert response.status_code == 200
+  #  mtcar.model_validate(response.json())
```

Comment:
```suggestion
```

Style: Commented-out blocks of code shouldn't make it to `main`. If this is a remaining "TODO" that isn't going to be addressed in this PR, then just open a new GH issue with a bit of context on what the TODO is (you can also copy this code chunk into that issue if you don't want to lose the work)